easybib/providers/crontab.rb

```diff
@@ -66,3 +66,31 @@
   new_resource.updated_by_last_action(updated)
 
 end
+
+action :delete do
+  app = new_resource.app
+
+  updated = false
+
+  execute 'Clear old cron.d files' do
+    # rm will exit with 1 if no old cron.d files existed
+    # adding a "; true" to remove the loooong warning in chef logs everyone stumbles upon
+    command "rm /etc/cron.d/#{app}_*; true"
+    ignore_failure true
+  end
+
+  execute 'Clear old crontab' do
+    updated = true
+    user node['nginx-app']['user']
+    # crontab will exit with 130 if crontab has already been cleared
+    # adding a "; true" to remove the loooong warning in chef logs everyone stumbles upon
+    command "crontab -u #{node['nginx-app']['user']} -r; true"
```

Comment:
Make an attribute for `node['nginx-app']['user']` and inject that value instead please.